src/Client/Mailer.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace BigGive\Identity\Client;
+
+use BigGive\Identity\Application\Settings\SettingsInterface;
+use GuzzleHttp\Client;
+use GuzzleHttp\Exception\GuzzleException;
+use GuzzleHttp\Exception\RequestException;
+use Psr\Log\LoggerInterface;
+
+class Mailer
+{
+    public function __construct(
+        private readonly SettingsInterface $settings,
+        private readonly LoggerInterface $logger,
+    ) {
+    }
+
+    public function sendEmail(array $requestBody): bool
+    {
+        try {
+            $httpClient = new Client([
```

Comment:
This should probably be a once-per-app-run service and use DI too. We don't want to unnecessarily spin up N connections to send N emails in a task for example.

This should also facilitate mocking the HTTP client itself in order to build a meaningful test for `Client\Mailer`, which feels like something we should ultimately be aiming to do as well as testing the more abstracted `Action` logic.